boot/modeenv.go

```diff
@@ -376,6 +376,27 @@ func (m *Modeenv) TryModelForSealing() secboot.ModelForSealing {
 	}
 }
 
+func (m *Modeenv) useThisModel(model *asserts.Model) {
+	m.Model = model.Model()
+	m.BrandID = model.BrandID()
+	m.Grade = string(model.Grade())
+	m.ModelSignKeyID = model.SignKeyID()
+}
+
+func (m *Modeenv) useThisTryModel(model *asserts.Model) {
```

Comment:
in the other PR I made some comments about the names of these

Reply:
renamed